### PR TITLE
Add DDPG-beta

### PR DESCRIPTION
**Pull request by ChildTang**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have ensured `make test` pass (**required**).
- [x] I have checked the code using `make format` (**required**).

### CODE REVIEW

**Comment by ChildTang** [comment]
Add DDPG algorithm

File: openrl/algorithms/dqn.py [hunk 1]
```diff
@@ -207,7 +206,7 @@ def prepare_loss(
         )
 
         q_targets = rewards_batch + self.gamma * max_next_q_values
-        q_loss = torch.mean(F.mse_loss(q_values, q_targets.detach()))  # 均方误差损失函数
+        q_loss = torch.mean(F.mse_loss(q_values, q_targets))  # 均方误差损失函数
```
Comment (huangshiyu13) [inline openrl/algorithms/dqn.py:209]: miss detach

File: examples/cartpole/train_dqn_beta.py [hunk 2]
```diff
@@ -13,27 +13,30 @@ def train():
...
-    env = make("CartPole-v1", render_mode="group_human", env_num=1, asynchronous=True)
+    env = make("CartPole-v1", render_mode="group_human")
     agent.set_env(env)
     obs, info = env.reset()
     done = False
     step = 0
     while not np.any(done):
         # 智能体根据 observation 预测下一个动作
-        action, _ = agent.act(obs)
+        action, _ = agent.act(obs["policy"])
```
Comment (huangshiyu13) [inline examples/cartpole/train_dqn_beta.py:39]: remove policy
